content/05.discussion.md

```diff
@@ -1,64 +1,72 @@
+Through our comprehensive analysis, we were able to identify how journalistic coverage varies by country, name origin, and gender, and compare it to scientific publishing background rates.
+While we found a significant gender disparity, the rate of female representation in scientific journalism is increasing and outpacing first and last authorships on scientific manuscripts.
+Furthermore, we identified a significant depletion of quotes from scientists with a predicted East Asian name origin when compared to primary research authorship, and a significant but smaller depletion of cited authors with a predicted East Asian name origin in journalistic content.
 Finally, we showed that coverage of specific countries differ in content, with the country's scientific output being put in a more significant focus for some countries than the environmental aspects of other countries.
 
 Previous anecdotal studies from journalists have shown that awareness of their bias can help them to reduce it [@https://medium.com/ladybits-on-medium/i-analyzed-a-year-of-my-reporting-for-gender-bias-and-this-is-what-i-found-a16c31e1cdf; @https://www.theatlantic.com/technology/archive/2016/02/gender-diversity-journalism/463023/; @https://www.theatlantic.com/science/archive/2018/02/i-spent-two-years-trying-to-fix-the-gender-imbalance-in-my-stories/552404/].
 Once a bias is identified an individual can seek resources to help them find and retain diverse sources, such as utilizing international expert databases like gage [@https://gage.500womenscientists.org/] and SheSource [@https://www.womensmediacenter.com/shesource/].
 Additional tips for journalists to achieve and maintain a diverse source pool is described by Christina Selby in the Open Notebook [@https://www.theopennotebook.com/2016/08/23/including-diverse-voices-in-science-stories/].
 
-While removing biases from their coverage should be a focus for editors and journalists, many journalists are limited by the persons who can respond to their requests for an interview or leads from prominent scientists.
+It should also be mentioned that we were only able to analyze the data provided through scraping "www.nature.com".
+This is a major limitation, because the only measures that we have of demographics of sources are persons who have their name mentioned within the article or cited in the article.
+This is only one aspect of the journalistic research process; a journalist may talk to several diverse persons, read their work, but not cite or mention them for several possible reasons.
+For example, a person may not be directly included in the article because of journalisitc style, that a person does not want to be included, or that they only provide informational content that is not directly quotable, but instead shapes the content of the article.
```

Comment:
```suggestion
For example, a person may not be mentioned or quoted included in the article because of length limitations, because they do not want to be named, or if they provide information that is not directly quotable but that still shapes the content of the article.
```